### DISCUSSION TITLE
i can't enable my github copilot

### DISCUSSION THREAD

**Discussion by VRyo** [community/community#58719] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I have followed the given steps and the congrats page appeared, but when I checked my account, Copilot is still not enabled, and I have to go through the payment steps again. I also checked my payment history, and there is no invoice for the previous payment I made. The money in my account has not been deducted. What should I do?

**Comment by davecheney** [comment]
@VRyo you have no active copilot subscription. Please visit https://github.com/github-copilot/signup to subscript to copilot.

**Comment by diegosaraviamamani** [comment]
I have the same issue, I visit the subscription url successfully but the subscription has not been done.

**Reply by davecheney** [reply]
@diegosaraviamamani you account is in error. I do not have the details. Please contact support https://support.github.com/contact
